internal/common/db.go

```diff
@@ -91,6 +91,15 @@ func GetClusterFromDB(db *gorm.DB, clusterId strfmt.UUID, eagerLoading EagerLoad
 	return c, nil
 }
 
+func GetAllClusterFromDB(db *gorm.DB, clusterId strfmt.UUID, eagerLoading EagerLoadingState) (*Cluster, error) {
```

Comment:
not sure why do you need this wrapper.

Reply:
to differ from getting only clusters that are registered.

Reply:
you can do the same with not found error